Amazon_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/basic_triggers/test_module1.py

```diff
@@ -42,6 +42,11 @@ async def test_route_add_del(testbed):
         )
         assert out[0][dev.host_name]["rc"] == 0, f"No Default route on {dev.host_name}"
         default_route = out[0][dev.host_name]["parsed_output"]
+
+        if not default_route or "gateway" not in default_route[0]:
+            print (f"Could not get the default route {out}")
```

Comment:
1.  Do you have an example print?
2.  Should `dev` be printed too?

Reply:
@muchetan can you please review this comment from Paul